scripts/tmux-ssh-split.sh

```diff
@@ -406,18 +399,19 @@ extract_path_from_ps1() {
       return 0
     fi
 
+    # Replace ~ with $HOME to avoid no such file or directory error. See issue #16
+    match="${match/\~/\$HOME}"
+
     echo -n "$match"
     return 0
   fi
 
   # Search for paths
-  if match=$(grep -m 1 -oP '/\K[^ ]*' <<< "$line")
+  if match=$(grep -m 1 -oP '/[^ ]*' <<< "$line")
   then
-    # Add leading slash if missing
-    [[ ! $match = /* ]] && match="/$match"
-    # Remove trailing '$', '#' and ']'
+    # Remove trailing '$', '#' and ']' See issue #17
     # Remove quotes (eg: ' or ")
-    sed -e 's/[]$#]$//' -e "s#['\"]*##g" <<< "${match}"
+    sed -r -e 's/[]$#]+$//' -e "s/['\"]//g"  <<< "${match}"
```

Comment:
0edeb5365e69221ba9d97e995e020b16a94dd98e

Reply:
You are missing a `+` in the first find replace.

Reply:
Can you please a pr? :)